include/boost/histogram/python/axis.hpp

```diff
@@ -85,7 +85,7 @@ py::array to_values_impl(const A &ax,
     for(auto i = -underflow; i < ax.size() + overflow; i++)
         result.mutable_at(static_cast<std::size_t>(i + underflow)) = ax.value(i);
 
-    return result;
+    return std::move(result);
```

Comment:
All pybind11 objects can be passed by value, because they derive from pybind11::handle, which is a smart pointer. Moving makes no sense here. Also in general, it may be detrimental to move, because it prevents NRVO.

Reply:
The compiler throws warnings here if you don't put move, since it can't copy elide here. I'll look into it more.

Reply:
Ok, nevermind, moving actually preserves the refcount. So one saves one decrement and one increment.

Reply:
Clang is very good with move warnings, it can also warning you when you are moving and could use copy elision instead.